experimental/Schemes/SpaceGerms.jl

```diff
@@ -63,37 +201,120 @@ end
     return Spec(R)
 end
 
-function point(X::SpaceGerm{<:Any, <:Any, <:GermAtClosedPoint})
+@doc raw"""
+    point(X::AbsSpaceGerm)
+
+Returns the point `p` of a germ `(X,p)`.
+
+# Examples:
+```jldoctest
+julia> R, (x,y,z) = QQ["x", "y", "z"];
+
+julia> I = ideal(R, [(x-1)*(x^2 - y^2 + z^2)]);
+
+julia> X = Spec(R, I);
+
+julia> XS = SpaceGerm(X,[0,0,0]);
+
+julia> point(XS)
+3-element Vector{QQFieldElem}:
+ 0
+ 0
+ 0
+
+```
+"""
+function point(X::AnySpaceGermClosedPoint)
   return point_coordinates(inverted_set(OO(X)))
 end
 
 ## currently unused case
-function point(X::SpaceGerm{<:Any, <:Any, <:GermAtGeometricPoint})
+function point(X::AnySpaceGermGeometricPoint)
   return prime_ideal(inverted_set(OO(X)))
 end
 
 Oscar.ring(X::AbsSpaceGerm) = OO(X)
 
-function Oscar.ideal(X::AbsSpaceGerm{<:Ring,<:MPolyQuoLocRing})
+function Oscar.ideal(X::AnySpaceGerm)
```

Comment:
I think we called this `defining_ideal`, for instance for projective varieties.

Reply:
done -- and I also fixed the analogue for coordinate_ring